package.json

```diff
@@ -10,7 +10,6 @@
   "private": true,
   "sideEffects": false,
   "scripts": {
-    "postinstall": "nr prisma:push",
```

Comment:
Let's prevent [pushing the database automatically upon running `pnpm i` command](https://app.warp.dev/block/cf9otHzTwznBCbL2dAJamA)